webserver/views/api.py

```diff
@@ -140,7 +142,10 @@ def _validate_auth_header():
 
     user = db.user.get_by_token(auth_token)
     if user is None:
-        user = db.user.create_user_with_token(auth_token)
+        if CREATE_FAKE_USERS_FOR_STRESS_TESTING:
```

Comment:
If you pinky swear that we're going to delete this code very soon, I'll let it slide, but a `DEBUG` guard would make me much less anxious